scripts/setup-socorro.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# create DB if it does not exist
+# TODO handle DB not on localhost - could use setupdb for this
+su - postgres -c "psql breakpad -c ''" > /var/log/socorro/setupdb.log 2>&1
```

Comment:
I don't want to embed ```sudo``` in the script, that seems too surprising - I'd rather the script just check if you're running as root and fail if not, then people can ```sudo``` or ```su``` as appropriate.